[Build system issue]: configure script doesn't check for appropriate `AR`

### What happened?

I tried cross compiling `shairport-sync` which actually works except for the following problem:

This is no wonder, as the generated `Makefile` (after configuring) contains the following line:

```
AR = ar
```

This is wrong, however, since the correct `AR` to use (in this instance) would have been `aarch64-unknown-linux-ar` which is the value of the `AR` environment variable (which gets overwritten in the `Makefile`). To work around the problem, passing `AR=aarch64-unknown-linux-ar` to `make(1)` works.

However, the configure script should just use a standard autoconf check for the `ar` to use which would make it discover the correct one. This works fine for other binutils, like `strip`.

### Relevant log output

```
autoreconf: export WARNINGS=
autoreconf: Entering directory '.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force
autoreconf: configure.ac: tracing
autoreconf: configure.ac: not using Libtool
autoreconf: configure.ac: not using Intltool
autoreconf: configure.ac: not using Gtkdoc
autoreconf: running: /nix/store/b39bm70xs1pkjay30b57yrxywxnx4fhg-autoconf-2.71/bin/autoconf --force
autoreconf: running: /nix/store/b39bm70xs1pkjay30b57yrxywxnx4fhg-autoconf-2.71/bin/autoheader --force
autoreconf: running: automake --add-missing --copy --force-missing
configure.ac:34: installing './compile'
configure.ac:6: installing './install-sh'
configure.ac:6: installing './missing'
Makefile.am: installing './INSTALL'
Makefile.am: installing './depcomp'
autoreconf: Leaving directory '.'
updateAutotoolsGnuConfigScriptsPhase
updateAutotoolsGnuConfigScriptsPhase
configuring
configure flags: --disable-dependency-tracking --prefix=/nix/store/k9ds27g2hsf8y5bi2nsr99cg2chi57pw-shairport-sync-aarch64-unknown-linux-gnu-4.2 --without-configfiles --sysconfdir=/etc --with-ssl=openssl --with-stdout --with-avahi --with-pa --with-pw --with-alsa --with-jack --with-stdout --with-pipe --with-soxr --with-dbus-interface --with-mpris-interface --build=x86_64-unknown-linux-gnu --host=aarch64-unknown-linux-gnu
checking for a BSD-compatible install... /nix/store/c2bq8xsayc90s33fd5xbm1vh5lrmwcfq-coreutils-9.3/bin/install -c
checking whether build environment is sane... yes
checking for aarch64-unknown-linux-gnu-strip... aarch64-unknown-linux-gnu-strip
checking for a race-free mkdir -p... /nix/store/c2bq8xsayc90s33fd5xbm1vh5lrmwcfq-coreutils-9.3/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for aarch64-unknown-linux-gnu-ranlib... aarch64-unknown-linux-gnu-ranlib
checking for git... no
checking for aarch64-unknown-linux-gnu-gcc... aarch64-unknown-linux-gnu-gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether aarch64-unknown-linux-gnu-gcc accepts -g... yes
checking for aarch64-unknown-linux-gnu-gcc option to enable C11 features... none needed
checking whether aarch64-unknown-linux-gnu-gcc understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of aarch64-unknown-linux-gnu-gcc... none
checking whether the compiler supports GNU C++... yes
checking whether aarch64-unknown-linux-gnu-g++ accepts -g... yes
checking for aarch64-unknown-linux-gnu-g++ option to enable C++11 features... none needed
checking dependency style of aarch64-unknown-linux-gnu-g++... none
checking pkg-config is at least version 0.9.0... yes
checking for clock_gettime in -lrt... yes
checking for pthread_create in -lpthread... yes
checking for exp in -lm... yes
checking for popt... yes
checking for libconfig... yes
checking for libssl,libcrypto... yes
checking for soxr... yes
checking for avahi_client_new in -lavahi-client... yes
checking for avahi_strerror in -lavahi-common... yes
checking for alsa... yes
checking for jack... yes
checking for libpulse >= 0.9.2... yes
checking for libpipewire-0.3 >= 0.3.24... yes
checking for gio-unix-2.0 >= 2.30.0... yes
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for vfork.h... no
checking for getopt_long.h... no
checking for arpa/inet.h... yes
checking for fcntl.h... yes
checking for limits.h... yes
checking for mach/mach.h... no
checking for memory.h... yes
checking for netdb.h... yes
checking for netinet/in.h... yes
checking for stdint.h... (cached) yes
checking for stdlib.h... (cached) yes
checking for string.h... (cached) yes
checking for sys/ioctl.h... yes
checking for sys/socket.h... yes
checking for sys/time.h... yes
checking for syslog.h... yes
checking for unistd.h... (cached) yes
checking for inline... inline
checking for int16_t... yes
checking for int32_t... yes
checking for int64_t... yes
checking for pid_t... yes
checking for size_t... yes
checking for ssize_t... yes
checking for uint16_t... yes
checking for uint32_t... yes
checking for uint64_t... yes
checking for uint8_t... yes
checking for working alloca.h... yes
checking for alloca... yes
checking for error_at_line... yes
checking for fork... yes
checking for vfork... yes
checking for working fork... cross
configure: WARNING: result yes guessed because of cross compilation
checking for working vfork... (cached) yes
checking for atexit... yes
checking for clock_gettime... yes
checking for gethostname... yes
checking for inet_ntoa... yes
checking for memchr... yes
checking for memmove... yes
checking for memset... yes
checking for mkfifo... yes
checking for pow... yes
checking for select... yes
checking for socket... yes
checking for stpcpy... yes
checking for strcasecmp... yes
checking for strchr... yes
checking for strdup... yes
checking for strerror... yes
checking for strstr... yes
checking for strtol... yes
checking for strtoul... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating scripts/shairport-sync.service-avahi
config.status: creating Makefile
config.status: creating scripts/shairport-sync
config.status: creating config.h
config.status: executing depfiles commands
building
build flags: -j4 SHELL=/nix/store/51sszqz1d9kpx480scb1vllc00kxlx79-bash-5.2-p15/bin/bash
gdbus-codegen --interface-prefix org.gnome --generate-c-code dbus-interface ./org.gnome.ShairportSync.xml
gdbus-codegen --interface-prefix org.mpris --generate-c-code mpris-interface ./org.mpris.MediaPlayer2.xml
touch mpris-interface.h
touch dbus-interface.h
make  all-am
make[1]: Entering directory '/build/source'
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o shairport.o shairport.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o rtsp.o rtsp.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o mdns.o mdns.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o common.o common.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o rtp.o rtp.c
common.c: In function 'rsa_apply':
common.c:814:5: warning: 'PEM_read_bio_RSAPrivateKey' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  814 |     rsa = PEM_read_bio_RSAPrivateKey(bmem, NULL, NULL, NULL);
      |     ^~~
In file included from common.c:77:
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/pem.h:447:1: note: declared here
  447 | DECLARE_PEM_rw_cb_attr(OSSL_DEPRECATEDIN_3_0, RSAPrivateKey, RSA)
      | ^~~~~~~~~~~~~~~~~~~~~~
common.c:818:3: warning: 'RSA_size' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  818 |   uint8_t *out = malloc(RSA_size(rsa));
      |   ^~~~~~~
In file included from /nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/x509.h:36,
                 from /nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/pem.h:23:
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/rsa.h:204:27: note: declared here
  204 | OSSL_DEPRECATEDIN_3_0 int RSA_size(const RSA *rsa);
      |                           ^~~~~~~~
common.c:821:5: warning: 'RSA_private_encrypt' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  821 |     *outlen = RSA_private_encrypt(inlen, input, out, rsa, RSA_PKCS1_PADDING);
      |     ^
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/rsa.h:285:5: note: declared here
  285 | int RSA_private_encrypt(int flen, const unsigned char *from, unsigned char *to,
      |     ^~~~~~~~~~~~~~~~~~~
common.c:824:5: warning: 'RSA_private_decrypt' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  824 |     *outlen = RSA_private_decrypt(inlen, input, out, rsa, RSA_PKCS1_OAEP_PADDING);
      |     ^
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/rsa.h:291:5: note: declared here
  291 | int RSA_private_decrypt(int flen, const unsigned char *from, unsigned char *to,
      |     ^~~~~~~~~~~~~~~~~~~
common.c:829:3: warning: 'RSA_free' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  829 |   RSA_free(rsa);
      |   ^~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/rsa.h:293:28: note: declared here
  293 | OSSL_DEPRECATEDIN_3_0 void RSA_free(RSA *r);
      |                            ^~~~~~~~
shairport.c: In function 'main':
shairport.c:2529:3: warning: 'MD5_Init' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 2529 |   MD5_Init(&ctx);
      |   ^~~~~~~~
In file included from shairport.c:64:
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:49:27: note: declared here
   49 | OSSL_DEPRECATEDIN_3_0 int MD5_Init(MD5_CTX *c);
      |                           ^~~~~~~~
shairport.c:2530:3: warning: 'MD5_Update' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 2530 |   MD5_Update(&ctx, config.service_name, strlen(config.service_name));
      |   ^~~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:50:27: note: declared here
   50 | OSSL_DEPRECATEDIN_3_0 int MD5_Update(MD5_CTX *c, const void *data, size_t len);
      |                           ^~~~~~~~~~
shairport.c:2531:3: warning: 'MD5_Update' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 2531 |   MD5_Update(&ctx, config.hw_addr, sizeof(config.hw_addr));
      |   ^~~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:50:27: note: declared here
   50 | OSSL_DEPRECATEDIN_3_0 int MD5_Update(MD5_CTX *c, const void *data, size_t len);
      |                           ^~~~~~~~~~
shairport.c:2532:3: warning: 'MD5_Final' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 2532 |   MD5_Final(ap_md5, &ctx);
      |   ^~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:51:27: note: declared here
   51 | OSSL_DEPRECATEDIN_3_0 int MD5_Final(unsigned char *md, MD5_CTX *c);
      |                           ^~~~~~~~~
rtsp.c: In function 'rtsp_auth':
rtsp.c:4889:3: warning: 'MD5_Init' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4889 |   MD5_Init(&ctx);
      |   ^~~~~~~~
In file included from rtsp.c:58:
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:49:27: note: declared here
   49 | OSSL_DEPRECATEDIN_3_0 int MD5_Init(MD5_CTX *c);
      |                           ^~~~~~~~
rtsp.c:4890:3: warning: 'MD5_Update' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4890 |   MD5_Update(&ctx, username, strlen(username));
      |   ^~~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:50:27: note: declared here
   50 | OSSL_DEPRECATEDIN_3_0 int MD5_Update(MD5_CTX *c, const void *data, size_t len);
      |                           ^~~~~~~~~~
rtsp.c:4891:3: warning: 'MD5_Update' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4891 |   MD5_Update(&ctx, ":", 1);
      |   ^~~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:50:27: note: declared here
   50 | OSSL_DEPRECATEDIN_3_0 int MD5_Update(MD5_CTX *c, const void *data, size_t len);
      |                           ^~~~~~~~~~
rtsp.c:4892:3: warning: 'MD5_Update' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4892 |   MD5_Update(&ctx, realm, strlen(realm));
      |   ^~~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:50:27: note: declared here
   50 | OSSL_DEPRECATEDIN_3_0 int MD5_Update(MD5_CTX *c, const void *data, size_t len);
      |                           ^~~~~~~~~~
rtsp.c:4893:3: warning: 'MD5_Update' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4893 |   MD5_Update(&ctx, ":", 1);
      |   ^~~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:50:27: note: declared here
   50 | OSSL_DEPRECATEDIN_3_0 int MD5_Update(MD5_CTX *c, const void *data, size_t len);
      |                           ^~~~~~~~~~
rtsp.c:4894:3: warning: 'MD5_Update' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4894 |   MD5_Update(&ctx, config.password, strlen(config.password));
      |   ^~~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:50:27: note: declared here
   50 | OSSL_DEPRECATEDIN_3_0 int MD5_Update(MD5_CTX *c, const void *data, size_t len);
      |                           ^~~~~~~~~~
rtsp.c:4895:3: warning: 'MD5_Final' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4895 |   MD5_Final(digest_urp, &ctx);
      |   ^~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:51:27: note: declared here
   51 | OSSL_DEPRECATEDIN_3_0 int MD5_Final(unsigned char *md, MD5_CTX *c);
      |                           ^~~~~~~~~
rtsp.c:4896:3: warning: 'MD5_Init' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4896 |   MD5_Init(&ctx);
      |   ^~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:49:27: note: declared here
   49 | OSSL_DEPRECATEDIN_3_0 int MD5_Init(MD5_CTX *c);
      |                           ^~~~~~~~
rtsp.c:4897:3: warning: 'MD5_Update' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4897 |   MD5_Update(&ctx, req->method, strlen(req->method));
      |   ^~~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:50:27: note: declared here
   50 | OSSL_DEPRECATEDIN_3_0 int MD5_Update(MD5_CTX *c, const void *data, size_t len);
      |                           ^~~~~~~~~~
rtsp.c:4898:3: warning: 'MD5_Update' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4898 |   MD5_Update(&ctx, ":", 1);
      |   ^~~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:50:27: note: declared here
   50 | OSSL_DEPRECATEDIN_3_0 int MD5_Update(MD5_CTX *c, const void *data, size_t len);
      |                           ^~~~~~~~~~
rtsp.c:4899:3: warning: 'MD5_Update' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4899 |   MD5_Update(&ctx, uri, strlen(uri));
      |   ^~~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:50:27: note: declared here
   50 | OSSL_DEPRECATEDIN_3_0 int MD5_Update(MD5_CTX *c, const void *data, size_t len);
      |                           ^~~~~~~~~~
rtsp.c:4900:3: warning: 'MD5_Final' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4900 |   MD5_Final(digest_mu, &ctx);
      |   ^~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:51:27: note: declared here
   51 | OSSL_DEPRECATEDIN_3_0 int MD5_Final(unsigned char *md, MD5_CTX *c);
      |                           ^~~~~~~~~
rtsp.c:4959:3: warning: 'MD5_Init' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4959 |   MD5_Init(&ctx);
      |   ^~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:49:27: note: declared here
   49 | OSSL_DEPRECATEDIN_3_0 int MD5_Init(MD5_CTX *c);
      |                           ^~~~~~~~
rtsp.c:4960:3: warning: 'MD5_Update' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4960 |   MD5_Update(&ctx, buf, 32);
      |   ^~~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:50:27: note: declared here
   50 | OSSL_DEPRECATEDIN_3_0 int MD5_Update(MD5_CTX *c, const void *data, size_t len);
      |                           ^~~~~~~~~~
rtsp.c:4961:3: warning: 'MD5_Update' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4961 |   MD5_Update(&ctx, ":", 1);
      |   ^~~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:50:27: note: declared here
   50 | OSSL_DEPRECATEDIN_3_0 int MD5_Update(MD5_CTX *c, const void *data, size_t len);
      |                           ^~~~~~~~~~
rtsp.c:4962:3: warning: 'MD5_Update' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4962 |   MD5_Update(&ctx, *nonce, strlen(*nonce));
      |   ^~~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:50:27: note: declared here
   50 | OSSL_DEPRECATEDIN_3_0 int MD5_Update(MD5_CTX *c, const void *data, size_t len);
      |                           ^~~~~~~~~~
rtsp.c:4963:3: warning: 'MD5_Update' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4963 |   MD5_Update(&ctx, ":", 1);
      |   ^~~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:50:27: note: declared here
   50 | OSSL_DEPRECATEDIN_3_0 int MD5_Update(MD5_CTX *c, const void *data, size_t len);
      |                           ^~~~~~~~~~
rtsp.c:4966:3: warning: 'MD5_Update' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4966 |   MD5_Update(&ctx, buf, 32);
      |   ^~~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:50:27: note: declared here
   50 | OSSL_DEPRECATEDIN_3_0 int MD5_Update(MD5_CTX *c, const void *data, size_t len);
      |                           ^~~~~~~~~~
rtsp.c:4967:3: warning: 'MD5_Final' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 4967 |   MD5_Final(digest_total, &ctx);
      |   ^~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:51:27: note: declared here
   51 | OSSL_DEPRECATEDIN_3_0 int MD5_Final(unsigned char *md, MD5_CTX *c);
      |                           ^~~~~~~~~
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o player.o player.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o alac.o alac.c
player.c: In function 'audio_packet_decode':
player.c:221:5: warning: 'AES_cbc_encrypt' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  221 |     AES_cbc_encrypt(buf, packet, aeslen, &conn->aes, iv, AES_DECRYPT);
      |     ^~~~~~~~~~~~~~~
In file included from player.c:60:
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/aes.h:66:6: note: declared here
   66 | void AES_cbc_encrypt(const unsigned char *in, unsigned char *out,
      |      ^~~~~~~~~~~~~~~
player.c: In function 'player_thread_func':
player.c:1928:5: warning: 'AES_set_decrypt_key' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
 1928 |     AES_set_decrypt_key(conn->stream.aeskey, 128, &conn->aes);
      |     ^~~~~~~~~~~~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/aes.h:54:5: note: declared here
   54 | int AES_set_decrypt_key(const unsigned char *userKey, const int bits,
      |     ^~~~~~~~~~~~~~~~~~~
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o audio.o audio.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o loudness.o loudness.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o activity_monitor.o activity_monitor.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o mdns_avahi.o mdns_avahi.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o audio_alsa.o audio_alsa.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o audio_jack.o audio_jack.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o audio_stdout.o audio_stdout.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o audio_pipe.o audio_pipe.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o audio_pa.o audio_pa.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o audio_pw.o audio_pw.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o metadata_hub.o metadata_hub.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o dacp.o dacp.c
metadata_hub.c: In function 'metadata_write_image_file':
metadata_hub.c:213:5: warning: 'MD5_Init' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  213 |     MD5_Init(&ctx);
      |     ^~~~~~~~
In file included from metadata_hub.c:59:
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:49:27: note: declared here
   49 | OSSL_DEPRECATEDIN_3_0 int MD5_Init(MD5_CTX *c);
      |                           ^~~~~~~~
metadata_hub.c:214:5: warning: 'MD5_Update' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  214 |     MD5_Update(&ctx, buf, len);
      |     ^~~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:50:27: note: declared here
   50 | OSSL_DEPRECATEDIN_3_0 int MD5_Update(MD5_CTX *c, const void *data, size_t len);
      |                           ^~~~~~~~~~
metadata_hub.c:215:5: warning: 'MD5_Final' is deprecated: Since OpenSSL 3.0 [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
  215 |     MD5_Final(img_md5, &ctx);
      |     ^~~~~~~~~
/nix/store/fw8c76ihg5m7nac9jnhd1xmr3c46fjhc-openssl-aarch64-unknown-linux-gnu-3.0.9-dev/include/openssl/md5.h:51:27: note: declared here
   51 | OSSL_DEPRECATEDIN_3_0 int MD5_Final(unsigned char *md, MD5_CTX *c);
      |                           ^~~~~~~~~
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o dbus-service.o dbus-service.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -fno-common -Wno-multichar -Wall -Wextra -Wno-clobbered -Wno-psabi -pthread -DSYSCONFDIR=\"/etc\"  -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o mpris-service.o mpris-service.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -pthread -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o lib_dbus_interface_a-dbus-interface.o `test -f 'dbus-interface.c' || echo './'`dbus-interface.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -pthread -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o lib_mpris_interface_a-mpris-interface.o `test -f 'mpris-interface.c' || echo './'`mpris-interface.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -pthread -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o tinyhttp/lib_tinyhttp_a-chunk.o `test -f 'tinyhttp/chunk.c' || echo './'`tinyhttp/chunk.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -pthread -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o tinyhttp/lib_tinyhttp_a-header.o `test -f 'tinyhttp/header.c' || echo './'`tinyhttp/header.c
aarch64-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I.    -pthread -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/gio-unix-2.0 -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include -I/nix/store/13jx3bhkgfxmfbb6fiqs44bj3qvb5vib-glib-aarch64-unknown-linux-gnu-2.76.3-dev/include/glib-2.0 -I/nix/store/ra67ydf8yrz06c61xdzn6np0gc1brna6-glib-aarch64-unknown-linux-gnu-2.76.3/lib/glib-2.0/include -D_REENTRANT -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/pipewire-0.3 -I/nix/store/9sscrcfmznn53mlh96xfn3f0181fcnna-pipewire-aarch64-unknown-linux-gnu-0.3.71-dev/include/spa-0.2 -I/nix/store/zr4bl6hn57hfi604q36shvv549gl5fim-popt-aarch64-unknown-linux-gnu-1.19/include -g -O2 -Wno-missing-field-initializers -c -o tinyhttp/lib_tinyhttp_a-http.o `test -f 'tinyhttp/http.c' || echo './'`tinyhttp/http.c
rm -f lib_tinyhttp.a
ar cr lib_tinyhttp.a tinyhttp/lib_tinyhttp_a-chunk.o tinyhttp/lib_tinyhttp_a-header.o tinyhttp/lib_tinyhttp_a-http.o
/nix/store/51sszqz1d9kpx480scb1vllc00kxlx79-bash-5.2-p15/bin/bash: line 1: ar: command not found
make[1]: *** [Makefile:844: lib_tinyhttp.a] Error 127
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/build/source'
```


### Configuration Information.

Not applicable.

### How did you install Shairport Sync?

A package manager (apt, apt install, yum, pkg, etc.)

### Check previous issues

- [X] Confirm

--- a/configure.ac
+++ b/configure.ac
@@ -33,6 +33,7 @@ esac
 # Checks for programs.
 AC_PROG_CC
 AC_PROG_CXX
+AC_CHECK_TOOL(AR, ar, :)
 AC_PROG_INSTALL
 
 PKG_PROG_PKG_CONFIG([0.9.0])

